metrics/prometheus.go

```diff
@@ -537,6 +543,26 @@ const (
 	containerEnvPrefix   = "container_env_"
 )
 
+// DefaultContainerLabels implements ContainerLabelsFunc. It exports the
+// container name, first alias, image name as well as all its env and label
+// values.
+func DefaultContainerLabels(container *info.ContainerInfo) map[string]string {
+	set := map[string]string{"id": container.Name}
```

Comment:
Let's just makes it clear in the godoc comment for now & make sure it's clear when used in kubernetes too.